docs/old_images.md

```diff
@@ -0,0 +1,23 @@
+# Building old Berrymill images
+
+The previous versions of the EBcL SDK used _kiwi-ng_ for building images. Using _kiwi-ng_ was a quite pragmatic choice, since it’s an established tool to build images from binary IT distribution packages, but it turned out that _kiwi-ng_ is not flexible enough to build typical embedded images. Starting from EBcL SDK 1.3 the new _make_ and _generator_ based builds are used. This approach has the advantage that it’s flexible enough for any imaginable build flow, and that the builds are much more efficient. Nevertheless, at least for EBcL 1.x, _kiwi-ng_ is still provided and supported. If you are at the very beginning of your development, we recommend switching to the new build flow, since it is more efficient, and _kiwi-ng_ support will be most likely dropped with the EBcL 2.x line.
```

Comment:
```suggestion
The previous versions of the EBcL SDK used _kiwi-ng_ for building images. Using _kiwi-ng_ was a quite pragmatic choice, since it’s an established tool to build images from binary IT distribution packages. Nevertheless, it turned out that _kiwi-ng_ is not flexible enough to build typical embedded images. Starting from EBcL SDK 1.3 the new _make_ and _generator_ based builds are used. This approach has the advantage that it’s flexible enough for any imaginable build flow, and that the builds are much more efficient. Nevertheless, at least for EBcL 1.x, _kiwi-ng_ is still provided and supported. If you are at the very beginning of your development, we recommend switching to the new build flow, since it is more efficient, and _kiwi-ng_ support will be most likely dropped with the EBcL 2.x line.
```